.github/workflows/deploy-tests.yml

```diff
@@ -2,6 +2,7 @@ name: Deploy tests
 
 on:
   pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
```

Comment:
What type(s) does this exclude?

Reply:
Yes, I should have added a comment.  
The first three types are the defaults and  `ready_for_the_review` is added on top of that so that the changelog test happens when the button "ready for review" is clicked.

Here is the full list:
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request